iiblib/iib_client.py

```diff
@@ -404,3 +405,72 @@ def rebuild_index(self, index_image):
 
     def health(self):
         raise NotImplementedError
+
+    def add_deprecations(
```

Comment:
@yashvardhannanavati, @chandwanitulsi: add_arches and build_tags can't be specified in /builds/add-deprecations. Should it allow adding timestamp tag by build_tags, not sure about add_arches. What do you think?

Reply:
I don't think we need `add_arches` for this endpoint. Thinking about it, I believe Pub needs `build_tags` for creating the timestamp tag to make sure the data is always available. I believe we should add it.

Reply:
I agree for having build_tags. I actually added that in the PR I am creating for the iib-worker part, I will add it to API also if its missing there. Thanks for the catch Emily :)